js/compiler/compiler.go

```diff
@@ -119,48 +123,81 @@ func (c *Compiler) Transform(src, filename string) (code string, srcmap []byte,
 		return
 	}
 
-	code, srcmap, err = c.babel.Transform(c.logger, src, filename)
+	code, srcmap, err = c.babel.transformImpl(c.logger, src, filename, c.COpts.SourceMapEnabled, inputSrcMap)
+	// fmt.Println(code)
 	return
 }
 
+// Options are options to the compiler
+type Options struct { // TODO maybe have the fields an exported and use the functional options pattern
+	CompatibilityMode lib.CompatibilityMode
+	SourceMapEnabled  bool
+	// TODO maybe move only this in the compiler itself and leave ht rest as parameters to the Compile
```

Comment:
I guess it depends of we want to do it *or* not. I kind of prefer the options struct as it seems more ... structured ( :wink: :wink: ). If we want to keep it I will drop the TODO, otherwise I will need to DO it.

Reply:
:+1: for options struct
Also, it seems `Compile` is called only in 3 places, in bundle.js and in initcontext.js: it doesn't look as if anything would need an overwrite of compiler options, etc. So I think it can stay simple. Unless there is an expectation of that changing..

Reply:
:+1: same for me for keeping it and removing the comment